src/stack_mitosis/planner.clj

```diff
@@ -52,19 +52,19 @@
        (map op/delete)))
 
 (defn transform
-  [prefix instance]
+  [alias-fn instance]
```

Comment:
❓ what is `instance` in this case?

Reply:
A given RDS database instance, as returned by `rds:DescribeInstances`.